apps/web/src/components/tweet/Timeline.tsx

```diff
@@ -105,8 +110,8 @@ export const Timeline = ({
       </Text>
 
       {refresh ? <Spinner position="absolute" top="15%" right="50%" /> : null}
```

Comment:
`{refresh && <Spinner position="absolute" top="15%" right="50%" />}` would be cooler